include/circt/Dialect/FIRRTL/OpExpressions.td

```diff
@@ -133,7 +133,7 @@ class BinaryPrimOp<string mnemonic, string resultTypeFunction,
        `(` type($lhs) `,` type($rhs) `)` `->` type($result)
   }];
 
-  let extraClassDeclaration = !cast<code>(!strconcat(!cast<string>([{
+  let extraClassDeclaration = !cast<string>(!strconcat(!cast<string>([{
```

Comment:
all these casts can be dropped now (either in this patch or as a follow-on cleanup) now.  The upstream change is that the "code" type got replaced with a string type, eliminating the back and forth.